src/test/java/com/zendesk/maxwell/MysqlIsolatedServer.java

```diff
@@ -32,6 +32,10 @@ public void boot(String xtraParams) throws IOException, SQLException, Interrupte
 		if ( xtraParams == null )
 			xtraParams = "";
 
+		// By default, MySQL doesn't run under root. However, in an environment like Docker, this is a reality.
+		// By checking the HOME env variable for "/root", we can identify whether we are the root user
+		boolean isRoot = System.getenv().get("HOME").equals("/root");
```

Comment:
hmmmm that's a bit shaky.  I guess it's just in the test env so ok, but could we not check for a DOCKER_ something env var?